packages/block-library/src/image/edit.native.js

```diff
@@ -469,6 +483,7 @@ export class ImageEdit extends React.Component {
 										retryMessage={ retryMessage }
 										url={ url }
 										width={ width }
+										imageShapeStyle={ styles[ className ] }
```

Comment:
Since this is an _image_ component, what do you think if we name this prop `shapeStyle` instead?

Reply:
Makes sense.

Reply:
Thanks for the change!